app/scripts/controllers/jobs.controller.js

```diff
@@ -72,16 +74,18 @@ angular.module('IguassuControllers').controller(
 		};
 
 		var updateJobListPeriodically = function () {
-			const INTERVAL_1_SECOND = 1000;
+			const INTERVAL_10_SECOND = 10000;
```

Comment:
I think that 10 seconds is a little high value, 5 seconds is not a good value?

Reply:
Done.